tex/artikel/ToiletPaper.tex

```diff
@@ -0,0 +1,15 @@
+\vspace{-6ex}
+\section{Toilet Paper}
+\vspace{-3ex}
+\begin{multicols*}{2}
+News und den Klatsch am Fachbereich sowie wichtige Veranstaltungen findet ihr im Toilet- Paper.
+
+Wie der Name es schon verrät findet ihr das Toilet- Paper auf dem stillen Örtchen (im KP Gebäude).  
+Herausgegeben wird das Toilet- Paper von uns (der Fachschaft Physik). Dabei erscheint es unregelmäßig - so oft, wie es eben etwas Wichtiges zu schreiben gibt. 
+Wie es sich für die Physik gehört, handelt es sich nicht einfach um eine Infozeitung, sondern um ein Paper - eine wissenschaftliche Fachzeitschrift. Dies ist erkennbar an den strengen Anforderungen der Redaktion sowie der hohen wissenschaftlichen Relevanz unserer Klatschmeldungen.

```

Comment:
```suggestion
Wie es sich für die Physik gehört, handelt es sich nicht einfach um eine Infozeitung, sondern um ein Paper – eine wissenschaftliche Fachzeitschrift. Dies ist erkennbar an den strengen Anforderungen der Redaktion sowie der hohen wissenschaftlichen Relevanz unserer Klatschmeldungen.
```

Gedankenstriche sind Halbgeviertstriche, nicht Viertelgeviertstriche („minus“).

Reply:
actually didn't know that